lib/fog/core.rb

```diff
@@ -40,7 +40,6 @@
 require File.expand_path('../core/hmac', __FILE__)
 require File.expand_path('../core/logger', __FILE__)
 require File.expand_path('../core/model', __FILE__)
-require File.expand_path('../core/mock', __FILE__)
```

Comment:
I believe that rather the removing you need to change the path to the new one like `fog/mock` no?